### PR TITLE
Add categories table in migration file to group competencies

### LINKED ISSUE #369: Categories migration table should be added
## Expected behaviour

The database schema will need to include a categories table that connects and groups competencies to be organized for different reflections.

The fields to include are:
- Label
- Description
- Image

## Details and resources

ERD
Rhizone LMS_revised

Migration 15.

## Checklist

- [x] This issue is linked to the appropriate project.
- [x] This issue has appropriate labels.

### PR DESCRIPTION
**Pull request by sheacronin**
## Proposed changes

Closes #369 . This pull request adds a categories table to the database with the following fields: id, label, description, image_url, created_at, and updated_at. This categories table will be used by the competencies table to group the competencies by category. image_url is the only optional column on the table, and the frontend should be able to handle a category without an image. This is the second pull request for this issue, now with the updated ES Lint checks from the newest main branch.

## Checklist

- [x] Are the issues being addressed linked to this PR?
- [x] Do all commit messages start with the issue number?
- [x] Are all code changes sufficiently tested?
- [x] Are there screenshots for UI changes?

### CODE REVIEW

**Review by daveytech — CHANGES_REQUESTED** [review]
Great! As there are no official testing files for migrations and we do not have a staging environment, one suggestion is for the personal responsible for the PR to add the code blocks of the output of their migration files to the PR. I know you've already tested it but for practice are you able to add the code block as a comment before you merge? Other than that it satisfies the issue.

**Review by Nazanin-Rezaei — APPROVED** [review]
Category table  with image_url optional feature and created and updated features set to current-timestamp looks functional.

File: db/migration/V015__create_categories_table.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE categories (
+  id BIGINT NOT NULL AUTO_INCREMENT,
+  label VARCHAR(255) NOT NULL,
+  description TEXT NOT NULL,
+  image_url VARCHAR(255),
```
Comment (achichikalova) [inline db/migration/V015__create_categories_table.sql:5]: Do we need to make it required because we decided on the categories' images?

**Comment by sheacronin** [comment]
Here is a screenshot of the migration file running successfully.
flyway_build

Here is what the database looks like with the new table and its columns.
db_tables

**Comment by daveytech** [comment]
@sheacronin Yes Screenshot works! Thank you

**Review by daveytech — APPROVED** [review]
Thanks for the screenshots!

**Review by achichikalova — APPROVED** [review]

**Pull request merged**